test/unit/signup/master_domains_builder_test.rb

```diff
@@ -4,23 +4,23 @@
 
 class Signup::MasterDomainsBuilderTest < ActiveSupport::TestCase
   test 'org_name is base for the subdomain' do
-    new_domains = generate_domains(org_name: master_account.org_name)
```

Comment:
I don't get how this is better. If `master_accounts` exists, then what we had before should work; if `master_account` does NOT exist, then this will fail anyway in `generate_domains `.

Reply:
right, sorry, i did this over an old version and then just fixed the rebase conflicts without realising it was already fixed 👍